### PR TITLE
[""Hot""fix]  Fixes MK58 Damage Multiplier, Adds Missing Frame, Fixes Barrel Color and Fixes Name

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
This PR gives the NT MK58 its 1.3 damage multiplier (still cant figure out how to fix fire rate, humon can do that) adds the alternate gray frame (Old NT Palette), Fixes the barrel sprites colors being inverted and fixes the names 

## Why It's Good For The Game
MK58 now performes like it should and has some bug fixes

## Testing
i spawned in, assembled some guns then shot them

## Changelog
:cl:
add: New NT MK58 Frame

fix: Gave the MK58 its 1.3 damage multiplier
imageadd: edited NT MK58 Barrel color
imagedel: deleted some icons and images
spellcheck: Changed MK58 names to be consistant with old versions
/:cl:

### CODE REVIEW

**Comment by Humonitarian** [comment]
Hello,

The solution to things like firerate is to put the modifiers in part/gun/modular/mechanism/pistol

**Comment by ValoTheValo** [comment]
> Hello,
> 
> The solution to things like firerate is to put the modifiers in part/gun/modular/mechanism/pistol

wouldnt this effect every pistol?

**Comment by Humonitarian** [comment]
> > Hello,
> > The solution to things like firerate is to put the modifiers in part/gun/modular/mechanism/pistol
> 
> wouldnt this effect every pistol?

It would affect all modular pistols. So only the mk58.

File: code/modules/projectiles/guns/projectile/modular/modular_MK58.dm [hunk 1]
```diff
@@ -31,19 +32,19 @@
...
+				return "NT HG [caliber] \"Mk58 S\"" //security
 			if("rubber")
-				return "NT HG [caliber] \"Mk-58 a\""
+				return "NT HG [caliber] \"Mk58 T\"" //tacticool
 			if("excelsior")
-				return "NT HG [caliber] \"Mk-58 e\""
+				return "NT HG [caliber] \"Mk58 M\"" //military
 			if("serbian")
-				return "NT HG [caliber] \"Mk-58 T\""
+				return "NT HG [caliber] \"Mk58 SA\"" //serbian arms
```
[the remark this answers is not in the document]
Comment (ValoTheValo) [reply code/modules/projectiles/guns/projectile/modular/modular_MK58.dm:43]: fixed

**Pull request merged**